Library/Formula/libcppa.rb

```diff
@@ -1,42 +1,82 @@
-require "formula"
-
 class Libcppa < Formula
+  # TODO: since libcppa has been renamed to CAF, this formula should eventually
+  # be renamed to 'caf.rb'.
   homepage "http://actor-framework.org/"
-  url "https://github.com/actor-framework/actor-framework/archive/0.11.0.tar.gz"
-  sha1 "202f2fd72a5af59d7ace6b7300df1fcc19f1857f"
+  url "https://github.com/actor-framework/actor-framework/archive/0.12.0.tar.gz"
+  sha256 "de927f53f5765489406d636dcf042bf3f281bc75ebfe4fce933dc1d9cff60481"
+  head "https://github.com/actor-framework/actor-framework.git"
 
-  # since upstream has rename the project to actor-framework (or libcaf in its
-  # pkgconfig file), we need to rename libcppa to libcaf in the future
+  # Since CAF does not produce development releases, we repurpose --devel to
+  # install from the 'develop' branch.
+  devel do
+    url "https://github.com/actor-framework/actor-framework.git",
+      :branch => "develop"
+    version "0.13-devel"
+  end
 
   bottle do
     cellar :any
-    sha1 "b0e9bef1983d561763e21539c9b9196d75e5a935" => :yosemite
-    sha1 "ed71bb57236d2aecf4e19e5044ca3af22969b5c5" => :mavericks
-    sha1 "8224fe20d5d4bd184a9b6c15ddd6143740ea23ca" => :mountain_lion
+    sha1 "672795a5d8b6a888752de0f4dc841ceb12c6a3b6" => :yosemite
   end
 
   depends_on "cmake" => :build
 
   needs :cxx11
 
-  option "with-opencl", "Build with OpenCL actors"
-  option "with-examples", "Build examples"
-  option "without-check", "Skip build-time tests (not recommended)"
-
-  def install
-    ENV.cxx11
+  option "with-static", "build as static and shared library"
+  option "with-static-only", "build as static library only"
+  option "with-opencl", "build with support for OpenCL actors"
+  option "with-cash", "build with cash (CAF shell)"
+  option "with-nexus", "build with nexus (central runtime inspection component)"
+  option "with-riac", "build with riac (runtime inspection and configuration)"
+  option "without-unit-tests", "skip unit tests (not recommended)"
 
-    args = %W[
-      --prefix=#{prefix}
-      --build-static
-    ]
+  # Because Homebrew does not support option values, we use one option per
+  # log-level values.
+  option "with-log-level-error", "build with log level ERROR"
+  option "with-log-level-warning", "build with log level WARNING"
+  option "with-log-level-info", "build with log level INFO"
+  option "with-log-level-debug", "build with log level DEBUG"
+  option "with-log-level-trace", "build with log level TRACE"
```

Comment:
I hear you on the component options. 

However, I would like to keep some form of choosing different log levels, as they have different performance implications. For example, while I enable errors or warnings on test systems, I keep CAF quiet on production systems. Also, when users send bug reports, we instruct them to use higher log-levels. Without the ability to change the output, users would have to switch to manual installs. Is there a way to accommodate this setting?